src/main/resources/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl/UploadedKeyStoreSource/config.jelly

```diff
@@ -55,6 +55,11 @@
         uploadedCertFileInput.onchange = fileOnChange.bind(uploadedCertFileInput);
       }
       function fileOnChange() {
+        // only trigger validation if the PKCS12 upload is selected
```

Comment:
as there is more than one option, only trigger this validation if this descriptor is chosen.